### DISCUSSION TITLE
Integration with Jira Server

### DISCUSSION THREAD

**Discussion by triley72** [atlassian/github-for-jira#377] [General]
Good afternoon.  I know in the notes it mentions the possibility of a release that would function with the server approach.  Is this something being actively worked on or is it months/years down the road?  Just curious and appreciate any feedback!

Tim

**Comment by djallits** [comment]
I too would be interested in making this work with Jira Server. That said, I am unfamiliar with the code in this repository, or what is typically required to make a Jira Cloud Integration work as a Server Integration. Has anyone done any discovery to this end? I am more than willing to chip in some PRs to help get us there.

**Reply by triley72** [reply]
Check out the two links below, they helped me to get setup with the Server instance (I did a lot more digging around after I submitted the original comment).  The second link is the most important as you dont actually use a "user" for the connection when you create your OATH token, but rather whatever the organization name is (i.e. https://github.com/someorgname).  Hope this helps.

https://confluence.atlassian.com/jirakb/jira-cloud-dvcs-accounts-configure-automatic-team-invites-779160866.html
https://confluence.atlassian.com/jirakb/add-github-organization-repositories-to-jira-software-815256707.html
